cupyx/scipy/ndimage/measurements.py

```diff
@@ -200,3 +203,282 @@ def _kernel_finalize():
         y[i] = j + 1;
         ''',
         'cupyx_nd_label_finalize')
+
+
+_ndimage_variance_kernel = cupy.ElementwiseKernel(
+    'T input, R labels, raw X index, int32 size, raw Z mean',
+    'raw Z out',
+    """
+    for (ptrdiff_t j = 0; j < size; j++) {
+      if (labels == index[j]) {
+        atomicAdd(&out[j], (input - mean[j]) * (input - mean[j]));
+        break;
+      }
+    }
+    """)
+
+
+_ndimage_sum_kernel = cupy.ElementwiseKernel(
+    'T input, R labels, raw X index, int32 size',
+    'raw Y out',
+    """
+    for (ptrdiff_t j = 0; j < size; j++) {
+      if (labels == index[j]) {
+        atomicAdd(&out[j], input);
+        break;
+      }
+    }
+    """)
+
+
+def _ndimage_sum_kernel_2(input, labels, index, out, batch_size=4):
```

Comment:
Should I use the un-batched version here as I dont think people will change internal mechanics of cupy and the externally for the unbatched version outperformed it? ( I kept it there to ensure reduced memory usage but for the current  results, the unbatched version doesnt seem to provide any benefits.)

Reply:
Yes, I suppose the un-batched version is simpler and enough here. Others can follow the discussion on this PR later.

Reply:
Yes, I suppose the un-batched version is simpler and enough here. Others can follow the discussion on this PR later.

Reply:
Maybe the batched version is helpful in case `use_kern=True`?

Reply:
yeah I kinda forgot about the memory issues of the unbatched version when people have to relly on use_kern=True, the batched vesion should stay as agreed upon before.